other/webpack.config.es6.js

```diff
@@ -107,6 +107,10 @@ const config = {
       'process.env': {
         'NODE_ENV': JSON.stringify(process.env.NODE_ENV)
       }
+    }),
+    new webpack.DefinePlugin({
```

Comment:
That's more a personal choice. For me it's visual since the use of those constants is different from the others.
I could see it argued that those should be added to "ON_DEV" and "ON_PROD"